frontend/src/helpers/formatBigNumbers.ts

```diff
@@ -0,0 +1,3 @@
+export function formatBigNumbers(bigNumber: string) {
+  return bigNumber.replace(",", "");
+}
```

Comment:
Could we move this part `new BigNumber(SOME_AMOUNT).toFormat(2)` into the helper rather than asking it to be the input? It will be easier to use the helper if we don't need to remember the specific format of the input.

We could also move it to `/helpers/formatAmount.ts` where we have other amount formatting helpers.

```suggestion
import BigNumber from "bignumber.js";

export function formatBigNumbers(number: string | number) {
  const bigNumber = new BigNumber(number).toFormat(2);
  return bigNumber.replace(",", "");
}
```

Reply:
Sure 👍🏽